.github/workflows/test.yml

```diff
@@ -0,0 +1,49 @@
+---
+name: Python Unittests
+
+on:
+  pull_request:
+    branches:
+      - develop
+  push:
+    branches:
+      - develop
+      - main
+  workflow_dispatch:
+
+jobs:
+  unittest:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.8', '3.9', '3.10']
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Setup config
+        run: cp config/.env.example config/.env.local
+      - name: Install Poetry
+        uses: snok/install-poetry@v1
```

Comment:
This all looks really complicated but if possible can we use the `make init` command.  The goal here is to treat the GitHub Actions like any other user so we can run these pipelines offline if needed (e.g. in the case that they're unavailable).

But let me know if you think this will be possible.

Reply:
This is the easiest way to install poetry itself in the build container.